TestCases/nonequilibrium/viscwedge/viscwedge.cfg

```diff
@@ -41,15 +41,16 @@ AOA= 0.0
 SIDESLIP_ANGLE= 0.0
 %
 % Free-stream pressure (101325.0 N/m^2 by default, only Euler flows)  
-FREESTREAM_PRESSURE= 101325.0
+FREESTREAM_PRESSURE= 10.0
```

Comment:
What motivates the drop in pressure here? 10.0 Pa seems low, does that correspond to condition of Re = 10000.

Reply:
I think @CatarinaGarbacz or @fmpmorgado can shed light here.

Reply:
that's exactly it @jtneedels , the conditions are the same as before, but in terms of pressure and temperatures, instead of pressure and reynolds, so nothing changed in reality

Reply:
this is on the basis of what we talked in the last meeting, that we no longer have reynolds initialization because we would need to implement iterative methods for this. I didn't make this decision (with @MarcoFossati ) out of "laziness", but more because reynolds initialization doesn't add any particular advantage, it's really just a matter of what variabeles we provide in the config.

but if people specifically want reynolds initialization back in the code, we can make it a TO-DO

Reply:
This makes sense, I think keeping initialization in terms of pressure/temperature is fine. I just wanted to make sure I understood the reason for the change.